zerver/lib/feedback.py

```diff
@@ -25,7 +26,7 @@ def has_enough_time_expired_since_last_message(sender_email: str, min_delay: flo
     return delay > min_delay
 
 def get_ticket_number() -> int:
-    num_file = '/var/tmp/.feedback-bot-ticket-number'
+    num_file = os.path.join(settings.DEPLOY_ROOT, '.feedback-bot-ticket-number')
```

Comment:
This doesn't work, because DEPLOY_ROOT changes on every deploy in production (so your numbers would reset on every deploy).

Reply:
Okay this now avoids the filesystem entirely and uses the server’s message ID.